recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -36,7 +36,8 @@ PLUGIN_WEBSERVER_PATH ?= "/var/www/"
 
 PACKAGECONFIG ?= " \
     ${WPE_SNAPSHOT} \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth',            'bluetooth', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'bluetoothcontrol',     'bluetoothcontrol', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'bluetoothcontrol',     'bluetoothremote', '', d)} \
```

Comment:
Please make the bluetoothcontrol dependent on bluetoothremote as a package (not at this level) and choose bluetoothcontrol and bluetoothremote as separated features

Reply:
I'd prefer we enable both remote and wifi when bluetooth is enabled, is there any reason why should seperate them?

Reply:
Same here, prefer we keep things limited to a machine feature for bluetooth.